wlauto/resource_getters/standard.py

```diff
@@ -380,11 +385,15 @@ def resolve_resource(self, resource):
                     if a['path'] == found:
                         return a
         elif resource.name == 'revent':
-            filename = '.'.join([resource.owner.device.name, resource.stage, 'revent']).lower()
-            for asset in assets:
-                pathname = os.path.basename(asset['path']).lower()
-                if pathname == filename:
-                    return asset
+            device_model = resource.owner.device.get_device_model()
+            wa_device_name = resource.owner.device.name
+            for name in [device_model, wa_device_name]:
+                if name:
```

Comment:
Rather than doing

    for name in list:
        if name:
            # large block

Generally, the fewer indentation levels three are, the better.
I'd go with

    for name in list:
         if not name:
              continue
         # large block